### PR TITLE
Add unknown length array

### PR DESCRIPTION
**Pull request by chakravarthi589**

### CODE REVIEW

File: src/Java.md [hunk 1]
```diff
@@ -173,7 +173,7 @@ It would be appreciated if there are any Java VTK experts who could convert any
 | -------------- | ---------------------- | ------------- | ------- |
 [FullScreen](/Java/Utilities/FullScreen) | vtkRenderWindow | Example to Display Actors in a Full Screen Render Window
 [TimerLog](/Java/Utilities/TimerLog) | vtkTimerLog | Example to Demonstrate Timer support and logging.
-[UnknownLengthArray](/Java/Utilities/UnknownLengthArray) | vtkFloatArray | |
+[UnknownLengthArray](/Java/Utilities/UnknownLengthArray) | vtkFloatArray | Dynamic, self-adjusting array of float
```
Comment (ajpmaclean) [inline src/Java.md]: It does not go here, it goes in the Array section.

**Comment by ajpmaclean** [comment]
These array examples should go in an Array section in Java.md, use Cxx.md to see how this is done. Why don't you do KnownLengthArray and then you can use the markdown files for the two examples?

**Comment by chakravarthi589** [comment]
OK I'll do it.

File: src/Java.md (hunk 1 above)
Comment (chakravarthi589) [reply src/Java.md]: I have updated.  Please check.

**Comment by chakravarthi589** [comment]
KnownLengthArray is added with changes in markdown file . Please review.

**Comment by ajpmaclean** [comment]
There is a conflicting file

**Comment by chakravarthi589** [comment]
Resolved.

File: src/Java.md [hunk 2]
```diff
@@ -181,6 +181,7 @@ It would be appreciated if there are any Java VTK experts who could convert any
 | Example Name | Classes Demonstrated | Description | Image |
 | -------------- | ---------------------- | ------------- | ------- |
 [UnKnownLengthArray](/Java/Utilities/UnKnownLengthArray) | vtkFloatArray | UnKnown Length Array
+[KnownLengthArray](/Java/Utilities/KnownLengthArray) | vtkFloatArray | Known Length Arra
```
Comment (ajpmaclean) [inline src/Java.md]: Arra should be Array

Comment (chakravarthi589) [reply src/Java.md]: updated.

**Comment by ajpmaclean** [comment]
Thanks.
Please take care, small things like this waste a lot of time and if they end up on the web site it looks unprofessional.

**Comment by chakravarthi589** [comment]
sure. I'll be much more careful.

**Pull request merged**